src/schema/rules/files/deriv/preprocessed_data.yaml

```
@@ -104,6 +104,13 @@ fmap_pepolar_common:
     space: optional
     description: optional
 
+fmap_pepolar_m0scan:
+  $ref: rules.files.raw.fmap.pepolar_m0scan
+  entities:
+    $ref: rules.files.raw.fmap.pepolar_m0scan.entities
```

I think schemacode might not be dereferencing `rules.files.raw.fmap.pepolar_m0scan` before `rules.files.deriv.preprocessed_data.fmap_pepolar_m0scan`, unfortunately.

```suggestion
    $ref: rules.files.raw.fmap.pepolar.entities
```